create/create.go

```diff
@@ -32,7 +33,7 @@ const (
 )
 
 func ReadTemplateFile(path string) ([]byte, error) {
-    return templates.ReadFile(path)
+	return templates.ReadFile(path)
```

Comment:
When do we need to use `fs.ReadFile` vs `fs.ReadTemplateFile`? Should it all be consistent?

Reply:
We should. I looked at this long and hard. This instance is only here because we don't have the embedded filesystem as this point. This is used in `serve` and the filesystem is created in `create`. The integration tests work this way, but perhaps I need to look into creating the embedded filesystem again for serve.